src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -120,8 +120,11 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
   private def startRunnableCalls(): State = {
     tryStartingRunnableCalls() match {
       case Success(entries) =>
-        if (entries.nonEmpty) tryStartingRunnableCalls()
-        goto(WorkflowRunning)
+        if (entries.nonEmpty) {
+          startRunnableCalls()
+        } else {
+          goto(WorkflowRunning)
```

Comment:
`tryStartingRunnableCalls` returns everything that it detected as 'runnable', so if a scatter block was runnable, we want to keep calling `startRunnableCalls` until it returns nothing.  Only if we were able to start everything would we set the workflow state to running (recursively), otherwise fail the workflow.